build-tests/x86/rawhide/test-image-systemd-boot/config.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+#======================================
+# Helpers
+#--------------------------------------
+# shellcheck disable=SC1091
+test -f /.kconfig && . /.kconfig
+
+set -ex
+
+declare kiwi_iname
+
+#======================================
+# Greeting...
+#--------------------------------------
+echo "Configure image: [$kiwi_iname]..."
+
+#======================================
+# Fix kernel-install
+#--------------------------------------
+# patch sent upstream and merged. This patching here can
+# be deleted as soon as the main distro has an update for
+# systemd available
+patch -p0 < kernel-install.patch
+rm -f kernel-install.patch
```

Comment:
This patch should no longer be needed. The upstreamed version is now in Rawhide.

Reply:
great, thanks. I'll give it a test after work today

Reply:
done, checking if the build test completes